pycbc/filter/qtransform.py

```diff
@@ -38,7 +38,7 @@
 from pycbc.types.timeseries import FrequencySeries, TimeSeries
 from numpy import fft as npfft
 
-__author__ = 'Hunter Gabbard <hunter.gabbard@ligo.org>'
+__author__ = 'Hunter Gabbard <hunter.gabbard@ligo.org>, Andrew Lundgren <andrew.lundgren@aei.mpg.de'
```

Comment:
We should take out the __author__ lines. Just add it to the top in a copyright notice.